index.html

```diff
@@ -91,7 +102,7 @@
             },
 	    "REST-IOT" : {
                   title: "RESTful Design for Internet of Things Systems"
-                , href: "https://tools.ietf.org/html/draft-irtf-t2trg-rest-iot-06"
+                , href: "https://datatracker.ietf.org/doc/html/draft-irtf-t2trg-rest-iot-06"
```

Comment:
Should we refer [the latest draft](https://datatracker.ietf.org/doc/html/draft-irtf-t2trg-rest-iot-11)?
The title is also changed to "Guidance on RESTful Design for Internet of Things Systems" from draft-08.